.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js Build With Yarn
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: Yarn Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    timeout-minutes: 3
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm i -g yarn
+      - run: yarn install
+      - name: Run TSLint
+        run: yarn run lint
+      - run: CI='' yarn run build
```

Comment:
The `CI=''` token here will dump GitHub's CI environment variable, which is set to true by default. If we don't do this, then the react build script will treat warnings as errors.

I don't like doing this, because ideally we'd have warning-free code, but we have some dependencies that generate build errors, as well as some old code that does so. So until we find a way to clean those up, we'll have to ignore build warnings. Any build errors will still cause this check to fail.

Reply:
I agree, We can look at that next sprint. The front-end needs to be updated. Synk is saying we have a lot of vulnerabilities in our 3rd part packages.